.github/workflows/test-suite.yml

```diff
@@ -184,7 +184,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v5
-      - uses: dtolnay/rust-toolchain@1.85
+      - uses: dtolnay/rust-toolchain@1.89
         with:
           profile: minimal
           toolchain: nightly-2024-07-09
```

Comment:
not new in this PR, but you can't set a toolchain on dtolay's action, so this just gets ignored